src/server/stat_job.c

```diff
@@ -458,11 +454,13 @@ status_subjob(job *pjob, struct batch_request *preq, svrattrl *pal, int subj, pb
 		attribute *attr = get_jattr(pjob, JOB_ATR_eligible_time);
 
 		oldeligflags = attr->at_flags;
-		ATR_UNSET(attr);
+		mark_jattr_not_set(pjob, JOB_ATR_eligible_time);
+		attr->at_flags |= ATR_MOD_MCACHE;
```

Comment:
isnt this handled inside mark_jattr_not_set()?

Reply:
Yes.